tests/python_client/milvus_client/test_milvus_client_insert.py

```diff
@@ -1342,5 +1344,768 @@ def test_milvus_client_insert_after_json_path_index(self, enable_dynamic_field,
+
+        self.upsert(client, collection_name, int32_rows, partial_update=True)
+        self.upsert(client, collection_name, vector_rows, partial_update=True)
+        result = self.query(client, collection_name, filter=default_search_exp,
+                   check_task=CheckTasks.check_query_results,
+                   output_fields=[default_vector_field_name, default_int32_field_name],
+                   check_items={exp_res: rows,
+                                "with_vec": True,
+                                "pk_name": default_primary_key_field_name})[0]
+        assert len(result) == default_nb
 
+        self.drop_collection(client, collection_name)
+        
+    @pytest.mark.tags(CaseLabel.L1)
+    @pytest.mark.skip(reason="Partial update not ready")
+    def test_milvus_client_partial_update_null_to_value(self):
+        """
+        Target: test PU can successfully update null to a value
+        Method:
+            1. Create a collection, enable nullable fields init null
+            2. Partial update nullable field
+            3. Query null field
+        Expected: Nullfield should have the same value as updated
+        """
+        # step 1: create collection with nullable fields init null
+        client = self._client()
+        schema = self.create_schema(client, enable_dynamic_field=False)[0]
+        schema.add_field(default_primary_key_field_name, DataType.INT64, is_primary=True, auto_id=False)
+        schema.add_field(default_vector_field_name, DataType.FLOAT_VECTOR, dim=default_dim)
+        schema.add_field(default_int32_field_name, DataType.INT32, nullable=True)
+        index_params = self.prepare_index_params(client)[0]
+        index_params.add_index(default_primary_key_field_name, index_type="AUTOINDEX")
+        index_params.add_index(default_vector_field_name, index_type="AUTOINDEX")
+        index_params.add_index(default_int32_field_name, index_type="AUTOINDEX")
+        collection_name = cf.gen_collection_name_by_testcase_name(module_index=1)
+        self.create_collection(client, collection_name, default_dim, schema=schema, 
+                               consistency_level="Strong", index_params=index_params)
+        rows = cf.gen_row_data_by_schema(nb=default_nb, schema=schema, skip_field_names=[default_int32_field_name])
+        self.upsert(client, collection_name, rows, partial_update=True)
+
+        # step 2: Partial update nullable field
+        new_value = np.int32(99)
+        new_rows = [{default_primary_key_field_name: row[default_primary_key_field_name], 
+                    default_int32_field_name: new_value} for row in rows]
+        self.upsert(client, collection_name, new_rows, partial_update=True)
+        
+        # step 3: Query null field
+        #self.load_collection(client, collection_name)
+        result = self.query(client, collection_name, filter=default_search_exp,
+                   check_task=CheckTasks.check_query_results,
```

Comment:
only "is null" or "is not null" can query for null values.  I think we should verify the field value from:
null -> not null, and not null to null